src/Microsoft.TestPlatform.CommunicationUtilities/SocketCommunicationManager.cs

```diff
@@ -354,8 +355,8 @@ private string TryReceiveRawMessage(CancellationToken cancellationToken)
                         && socketException.SocketErrorCode == SocketError.TimedOut)
                     {
                         EqtTrace.Info(
-                            "SocketCommunicationManager ReceiveMessage: failed to receive message because read timeout {0}",
-                            ioException);
+                        "SocketCommunicationManager ReceiveMessage: failed to receive message because read timeout {0}",
```

Comment:
nit: not required change?

Reply:
Hey, It was already there. 


---
In reply to: [137164236](https://github.com/Microsoft/vstest/pull/1041#discussion_r137164236) [](ancestors = 137164236)

Reply:
Yeah, added the tabs back.